docs/api-docs/storefront/scripts-overview.mdx

```diff
@@ -36,6 +36,7 @@ These steps can help prevent unexpected behavior.
 - Each app can have 10 API-created scripts
 - Each channel can have 50 control panel-created scripts 
 - You can install up to five scripts in a single call
+- Scripts have a 20-second cache time on the storefront
```

Comment:
- On a BigCommerce-hosted storefront, new or updated scripts take 20 seconds to cache